includes/events/class-mdjm-event.php

```diff
@@ -1079,6 +1080,14 @@ public function playlist_is_enabled() {
 		return apply_filters( 'mdjm_playlist_status', $return, $this->ID );
 	} // is_playlist_enabled
 
+	/** Get the playlist limit
+	 * @return number
+	 */
+	public function get_playlist_limit() {
+		$pl_limit = get_post_meta( $this->ID, '_mdjm_event_playlist_limit', true );
```

Comment:
Highly recommend only pulling this info in one place so for this method simply use the following and utilise the function you created...

    public function get_playlist_limit() {
        return mdjm_get_playlist_limit( $this->ID );
    } // get_playlist_limit